results/feeds.py

```diff
@@ -19,24 +24,31 @@ class BasicResultEventsFeed(Feed):
     description = _("A basic feed of election results")
 
     def items(self):
-        return ResultEvent.objects.filter(created__gte="2017-05-04")
+        date_limit = datetime.now() - timedelta(days=14)
+        return ResultEvent.objects.filter(created__gte=date_limit) \
```

Comment:
My only concern about that is that if someone switches the election to `current=False` the next day or when the polls close, feed consumers could miss results, whereas a time window seems to fit the periodic checking behaviour of feed readers better. I'm happy to change it if you prefer `election__current=True`, though? (I'm not sure what our policy is on when to switch `current` to `False`, to be honest.)